hedera-node/src/main/java/com/hedera/services/txns/file/FileCreateTransitionLogic.java

```diff
@@ -70,14 +70,16 @@ public FileCreateTransitionLogic(
 
 	@Override
 	public void doStateTransition() {
+		/* --- Extract from gRPC --- */
 		var op = txnCtx.accessor().getTxn().getFileCreate();
 
+		/* --- Perform validations --- */
 		validateFalse(op.hasKeys() && !validator.hasGoodEncoding(wrapped(op.getKeys())), INVALID_FILE_WACL);
-
 		var attr = asAttr(op);
```

Comment:
I think we could use another comment like /* --- Extract variables --- */ to visually separate the extraction from the validation step.